src/lib/utils/meta_tables/meta_temporary_memory_usage_table.cpp

```diff
@@ -0,0 +1,35 @@
+#include "meta_temporary_memory_usage_table.hpp"
+
+#include <magic_enum.hpp>
+
+#include "hyrise.hpp"
+
+namespace opossum {
+
+MetaTemporaryMemoryUsageTable::MetaTemporaryMemoryUsageTable()
+    : AbstractMetaTable(TableColumnDefinitions{{"operator_type", DataType::String, false},
+                                               {"operator_data_structure", DataType::String, false},
+                                               {"timestamp", DataType::Long, false},
+                                               {"amount", DataType::Long, false}}) {}
+
+const std::string& MetaTemporaryMemoryUsageTable::name() const {
+  static const auto name = std::string{"memory"};
+  return name;
+}
```

Comment:
@tjjordan I wonder if we should rename the table to `"temporary_memory_usage"` to be consistent with the updated class name `MetaTemporaryMemoryUsageTable`.